server/build-windows-x64.sh

```diff
@@ -275,7 +275,7 @@ our \$config = {
     asserts=>0,                         # --enable-cassert
     nls=>'$PG_PGBUILD_GETTEXT_WINDOWS_X64',        # --enable-nls=<path>
     perl=>'$PG_PERL_WINDOWS_X64',             # --with-perl
-    python=>'$PG_PYTHON_WINDOWS_X64',         # --with-python=<path>
+    py=>'$PG_PYTHON_WINDOWS_X64',         # --with-python=<path>
```

Comment:
typo? py->python

Reply:
No typo, this is how the name of the python executable file is inside Python directory.

Reply:
fixed